samples/BluetoothSensorTag/sensorTagComponent/sensorTag.c

```diff
@@ -1486,6 +1571,14 @@ static void HandleIOPush
 
 COMPONENT_INIT
 {
+    // Simple Keys
+    LE_ASSERT_OK(io_CreateInput(RES_PATH_SIMPLE_KEYS_USER_VALUE, IO_DATA_TYPE_BOOLEAN, ""));
+    io_SetBooleanDefault(RES_PATH_SIMPLE_KEYS_USER_VALUE, false);
+    LE_ASSERT_OK(io_CreateInput(RES_PATH_SIMPLE_KEYS_POWER_VALUE, IO_DATA_TYPE_BOOLEAN, ""));
+    io_SetBooleanDefault(RES_PATH_SIMPLE_KEYS_POWER_VALUE, false);
+    LE_ASSERT_OK(io_CreateInput(RES_PATH_SIMPLE_KEYS_REED_VALUE, IO_DATA_TYPE_BOOLEAN, ""));
+    io_SetBooleanDefault(RES_PATH_SIMPLE_KEYS_REED_VALUE, false);
```

Comment:
Why do these need default values?

Reply:
Removed them in 67beb64f417a15060333ff615f32f6a7ed784315